BAC0/core/functions/Calendar.py

```diff
@@ -125,7 +107,7 @@ def send_calendar_request(self, request, timeout=10):
             if not isinstance(apdu, SimpleAckPDU):  # expect an ACK
                 self._log.warning("Not an ack, see debug for more infos.")
                 self._log.debug(
-                    "Not an ack. | APDU : {} / {}".format((apdu, type(apdu)))
+                    "Not an ack. | APDU : {} / {}".format(apdu, type(apdu))
```

Comment:
there's an extra set of parenthesis here that break formatting and raise an error. this bit of code is repeated a couple times so I won't call out of the rest of them